compiler/aarch64/codegen/OMRMemoryReference.hpp

```diff
@@ -59,7 +60,8 @@ class OMR_EXTENSIBLE MemoryReference : public OMR::MemoryReference
    TR::Node *_baseNode;
    TR::Register *_indexRegister;
    TR::Node *_indexNode;
-   int32_t _offset;
+   intptr_t _offset;
+   uint32_t _length;
```

Comment:
It seems `_length` is used nowhere.

Reply:
It is used in `getScaleForNode` and `populateMemoryReference`.

Reply:
I overlooked the use of `_length` in OMRMemoryReference.cpp.